FirstProject/FirstNumber.cs

```diff
@@ -0,0 +1,21 @@
+﻿
+namespace FirstProject
+{
+    internal class FirstNumber
+    { 
+        public void PrimeNumber(int number3)
```

Comment:
Metoda powinna być raczej boolem

Reply:
int number3? Co to za nazwa zmiennej, raczej bym się skłaniał ku value albo number